test/proposal.test.ts

```diff
@@ -54,6 +60,12 @@ describe("proposal", function () {
     multiTokenMediatorETH = await OmniBridgeTransferSimulator.connect(
       deployer,
     ).deploy();
+    const IAMB = await artifacts.readArtifact("IAMB");
+    arbitraryMessageBridge = await waffle.deployMockContract(
+      deployer,
+      IAMB.abi,
+    );
+    await arbitraryMessageBridge.mock.requireToPassMessage.returns(messageID);
```

Comment:
I think there should be a test that checks the input of `requireToPassMessage` somewhere, even if it copies some of the code use to generate the bytecode. I see that the input is manually decoded later, so maybe we could move the mock there and use the decoded values.

Reply:
Yes, this was a good point! I am not sure about best practice, but now we are testing:
- (1) the AMB will be called with certain parameters
- (2) thse parameters from (1) are executable and generate the correct safe

I think with that we are safe now